### PR TITLE
Fix button

### PR DESCRIPTION
**Pull request by martahj**
* Imports react so the button component will not error on being imported in a third party library.
* Bumps the version
* Prettier changes

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -19,23 +20,26 @@ const raised = css`
...
+    props.accent &&
+    `
     color: white;
     background-color: ${props.theme.accent};
     &:before {
       color: black;
     }
-  `}
-  ${props => props.primary && `
+  `} ${props =>
```
Comment (brad-decker) [inline src/components/Button.js]: this looks really weird. I assume this is a prettier thing but i dislike this format a lot.

File: src/components/Button.js [hunk 2]
```diff
@@ -61,9 +65,7 @@ const Button = styled(ButtonComponent)`
   user-select: none;
   box-sizing: border-box;
   -webkit-appearance: none;
-  ${elevationTransition}
-
-  &:active {
+  ${elevationTransition} &:active {
```
Comment (AriLFrankel) [inline src/components/Button.js]: Are the &active styles being applied without a semicolon after elevationTransition?

File: src/components/Button.js [hunk 3]
```diff
@@ -19,23 +20,26 @@ const raised = css`
   &:active {
     ${elevation(8)};
   }
-  ${props => props.accent && `
+  ${props =>
+    props.accent &&
+    `
```
Comment (brad-decker) [inline src/components/Button.js]: Also don't like this... i'm not sure what rule this is that is doing that but the line as it was written is fine.

File: src/components/Button.js [hunk 4]
```diff
@@ -76,23 +78,22 @@ const Button = styled(ButtonComponent)`
...
   }
 
-  ${ripple()}
-
-  ${props => props.accent && accent}
-  ${props => props.primary && primary}
-  ${props => props.raised && raised}
-  ${props => props.compact && `padding: 0 8px;`}
-  ${props => props.dense && `
+  ${ripple()} ${props => props.accent && accent} ${props =>
```
Comment (brad-decker) [inline src/components/Button.js]: what?

Comment (martahj) [reply src/components/Button.js]: Hmm.. you're right, this doesn't seem right.

File: src/components/Button.js [hunk 5]
```diff
@@ -19,23 +20,26 @@ const raised = css`
...
+      props.primary &&
+      `
     color: white;
     background-color: ${props.theme.primary};
     &:before {
       color: black;
     }
-  `}
-  fieldset:disabled &, &:disabled {
+  `} fieldset:disabled &,
```
Comment (AriLFrankel) [inline src/components/Button.js]: Is this style being applied?

**Comment by martahj** [comment]
We don't have prettier in this repo - the changes are coming from the "fix on save" feature of my editor.

**Comment by brad-decker** [comment]
It has to be hooked into prettier somehow because maybe you have it installed globally and fixOnSave taps into it?

**Comment by martahj** [comment]
@brad-decker @AriLFrankel Got rid of horrible prettier formatting

**Pull request merged**